# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xf86-video-trident/hotfix-ramdac.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-video-trident-1.3.8/src/trident.h.vanilla	2022-08-17 13:31:20.385667159 +0200
+++ xf86-video-trident-1.3.8/src/trident.h	2022-08-17 13:31:40.716665947 +0200
@@ -38,7 +38,7 @@
 #include "xaa.h"
 #endif
 #include "xf86fbman.h"
-#include "xf86RamDac.h"
+#include "xf86Cursor.h"
 #include "compiler.h"
 #include "vgaHW.h"
 #include "xf86i2c.h"
@@ -169,7 +169,6 @@
     CARD32		BltScanDirection;
     CARD32		DrawFlag;
     CARD16		LinePattern;
-    RamDacRecPtr	RamDacRec;
     int			CursorOffset;
     xf86CursorInfoPtr	CursorInfoRec;
     xf86Int10InfoPtr	Int10;
